### PR TITLE
Simplify get started very condenced

### PR DESCRIPTION
**Pull request by mike0sv**
https://mlem-ai-simplify-get-st-gq4xqx.herokuapp.com/doc/get-started

This is a draft for a draft.
Basically I got rid of some texts that tried to explain what is going on

### CODE REVIEW

**Comment by mike0sv** [comment]
One thing I noticed with toggles and tabs - if you have the same name for tabs in different toggles, you are switching all of them. Because tabs can have different length, you'll get jumpy content (try cli/api toggle for serving)

File: content/docs/command-reference/build.md [hunk 1]
```diff
@@ -34,5 +34,5 @@ images.
 
 ## Examples
 
-For examples, please refer to [Get Started](/doc/get-started/building) or
-[User Guide](/doc/user-guide/building).
+For examples, please refer to the
+[Building User Guide](/doc/user-guide/building).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/build.md:38]:
```suggestion
[Building](/doc/user-guide/building) guide.
```

File: content/docs/command-reference/deployment/index.md [hunk 2]
```diff
@@ -31,14 +31,11 @@ automatically used by MLEM to wire and expose endpoints on the application
...
 ## Options
 
 - `-h`, `--help` - Show this message and exit.
 
 ## Examples
 
-For examples, please refer to [Get Started](/doc/get-started/deploying) or
-[User Guide](/doc/user-guide/deploying).
+For examples, please refer to [Get Started](/doc/get-started) tutorial or the
+[Deploying User Guide](/doc/user-guide/deploying).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/deployment/index.md:41]:
```suggestion
[Deploying](/doc/user-guide/deploying) guide.
```

File: content/docs/command-reference/serve.md [hunk 3]
```diff
@@ -40,5 +40,5 @@ built-in client, or common HTTP clients, such as [`curl`](https://curl.se/) and
 
 ## Examples
 
-For examples, please refer to [Get Started](/doc/get-started/serving) or
-[User Guide](/doc/user-guide/serving).
+For examples, please refer to [Get Started](/doc/get-started) tutorial or the
+[Serving User Guide](/doc/user-guide/serving).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/serve.md:44]:
```suggestion
[Serving](/doc/user-guide/serving) guide.
```

File: content/docs/object-reference/mlem-abcs.md [hunk 4]
```diff
@@ -270,7 +270,7 @@ Clients for corresponding servers
 ### Builder
 
 Declaration for creating a `build` from model. You can learn more about building
-[here](/doc/get-started/building)
+in this [User Guide](/doc/user-guide/building)
```
Comment (jorgeorpinel) [inline content/docs/object-reference/mlem-abcs.md:273]:
```suggestion
in [this guide](/doc/user-guide/building)
```

File: content/docs/object-reference/model/index.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ using `pickle`, a special extension for that framework will allow to save the
 model in framework-specific format and support special features of that
 framework.
 
-Typicaly model extension will implement
+Typicaly model extensions will implement
```
Comment (jorgeorpinel) [inline content/docs/object-reference/model/index.md:13]:
```suggestion
Typicaly, model extensions will implement
```

File: content/docs/user-guide/building/index.md [hunk 6]
```diff
@@ -23,10 +23,11 @@ can run `mlem build --help` for list of builders and
 
 ## Pre-configured builders
 
-In [Get Started](/doc/get-started/building) we demonstrated how to build a
-docker image out of the model. Now let's see what is the builder declaration we
-mentioned there. You can pre-configure your builder in the form of yaml file
-that we call "declaration" either manually or via `mlem declare` command:
+In the [Get Started Tutorial](/doc/get-started) we demonstrated how to build a
```
Comment (jorgeorpinel) [inline content/docs/user-guide/building/index.md:26]: See https://github.com/iterative/mlem.ai/pull/257

File: content/docs/install.md [hunk 7]
```diff
@@ -10,7 +10,7 @@ MLEM is a Python library. You can install it with a package manager like
 [Conda](https://docs.conda.io/en/latest/), or as a Python
 [requirement](https://pip.pypa.io/en/latest/user_guide/#requirements-files).
 
-<admon type="info">
+<admon type="tip">
 
 We **strongly** recommend creating a [virtual environment] or using [pipx] to
```
Comment (jorgeorpinel) [inline content/docs/install.md:15]: Good idea. We should apply this to https://mlem.ai/doc/contributing/core#development-environment and https://mlem.ai/doc/gto/install#install-as-a-python-library as well. Minor

**Review by jorgeorpinel — COMMENTED** [review]
Some copy edits...

**Comment by jorgeorpinel** [comment]
This needs a merge from `main`, actually 🙂

**Comment by omesser** [comment]
@jorgeorpinel  - I'm going to use this to grab some ideas and put on top of the current get-started iterations. Not sure which ones yet, but I plan to then make a more "review ready" PR and will ping you there, let's keep this here just for me to use as a reference (sorry about the confusion this may cause).
BTW there are some things here that I disagree or would like to tweak and not take as is exactly.
I think that, among other things, @mike0sv @aguschin and myself (and maybe you as well) have a slightly different mental model of the HL workflow, and we need to reconcile that conceptually.

**Comment by jorgeorpinel** [comment]
This one def. looks stale. Probably safe to close (can always be reopened).

**Pull request closed without merging**